src/Traits/Hooks/UseDashboardZoneTwo.php

```diff
@@ -32,6 +32,12 @@ trait UseDashboardZoneTwo
      */
     public function hookDashboardZoneTwo(array $params)
     {
+        $this->context->smarty->assign(
+            [
+                'host_mode' => defined('_PS_HOST_MODE_'),
```

Comment:
checking that it's defined is enough or the value is important ?

`defined('_PS_HOST_MODE_') && (true === bool('_PS_HOST_MODE_))`
OR
`defined('_PS_HOST_MODE_') ? _PS_HOST_MODE_ : false`

Reply:
As I understand, it's defined only when it's true.
Furthermore, it seems it does not exist anymore at all...

Reply:
👍🏾